docs/helm-chart-checks.md

```diff
@@ -62,22 +62,22 @@ This section provides help on the basic usage of Helm chart checks with the podm
 - Run all the available checks for a remotely available chart using a `uri`, assuming the kube config file is available in ${HOME}/.kube:
 
   ```
-  $ podman run --rm -i \
-          -e KUBECONFIG=/.kube/config \
-          -v "${HOME}/.kube":/.kube \
+  $ podman run --rm -i                                  \
```

Comment:
Yes that's true. It's only for the sake of code readability and sytling. As https://github.com/redhat-certification/chart-verifier/pull/225 raised up my attention. I've seen this convention before and though this would be an improvement to readability. I'm totally okay to revert this since there's no functional difference.

Reply:
if you copy and paste the command, does it still work? make sure you are not braking the copy paste experience

Reply:
Yes, it works as before.